### DISCUSSION TITLE
Custom CSS No Longer Working

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#1253] [Q&A] [answered]
None of the custom CSS is working anymore. All has reverted back to default. Examples include adding our own  overlay image for online only meetings instead of the stock one used, changing the color from muted gray to green for the small text (e.g. women, men), as well as adding a forward slash. 

I don't know when this happened as I am not actively supporting the website anymore, but I have verified the CSS was not removed or changed.

Site: sussexaa.org

**Comment by joshreisner** [comment]
yes we simplified the CSS required to customize the online meeting background image as part of [this update](https://discourse.tiaa-forum.org/t/help-us-test-the-next-version-of-tsml-ui/9180/3)

here is the updated CSS: https://github.com/code4recovery/tsml-ui#change-the-online-background-image

**Reply by Grateful-Viv** [reply]
I see that now for the image overlay. And I will advise how to do this, but how can this also be done for what was 
```
#tsml-ui h1 small.text-muted::before
#tsml-ui h1 small.text-muted 
#tsml-ui .name small.text-muted
#tsml-ui .name small.text-muted::before 

```
And instead of a gray box for inactive meetings, we had the following:
#tsml-ui .meeting.inactive .map::after {
   content: "Inactive Meeting";
    display: block;
    visibility:visible; 
    border: 5px solid #D40047;
    opacity: 0.85;
    background:#fff;
    color: #D40047;
    font-size: 25px;
    font-weight: bold;
    text-align: center;
    padding: 20px;
    position: absolute;
    z-index: 2;
    top: 0;
    left: 0;
    right: 0;
    bottom: 0;
}

This is all styling and none of it affects the functionality of TSML, so if it's too big a deal to change, I'll let it go.

Thank you.

**Reply by joshreisner** [reply, marked as answer]
the recent style change made some things easier, like matching the fonts and colors to a website theme, but also discourages more unique customizations in favor of encouraging a more consistent experience across sites.

i think if there's a problem, like if something is confusing to users, then it's probably something we want to fix for all sites.

**Reply by Grateful-Viv** [reply]
The styles I had put in were more consistent with how TSML used to be about 4 years ago. 

Instead of a faded grey color for women or men or open or closed, it had a forward slash and was in a blue-green color.  Inactive meetings had the text: "Meeting Inactive" in bold red after the meeting name, along with a box with a red border and the words "Inactive Meeting" inside of it, when you opened the detail page.  

I didn't have as much opacity for the background online image. That was a subjective style change. I also didn't show the image when it was viewed on phones. Again, subjective style change.

There were a host of other style changes, but with the CSS simplification update, none of these work.  

I hear you about wanting it to be uniform and consistent across all sites, so I guess I am going to drop this, especially since I am no longer actively supporting the site.
